sensu/plugins/check-vyatta-tunnels.py

```diff
@@ -59,11 +59,11 @@ def main():
         switch_on_criticality()
 
     for index, device in enumerate(device_list):
-        if " vti" in str(device):
```

Comment:
Look at the defect 334196 Description (I won't access customer environment, so just capture these data). ip_vti0 is down. we won't discover it if there is a space on front of vti0.
iso.3.6.1.2.1.2.2.1.2.10 = STRING: "ip_vti0"
iso.3.6.1.2.1.2.2.1.2.11 = STRING: "vti0"

iso.3.6.1.2.1.2.2.1.8.9 = INTEGER: 1
iso.3.6.1.2.1.2.2.1.8.10 = INTEGER: 2